tests/QAN/pages/qanDetailsFragment.js

```diff
@@ -61,9 +67,16 @@ module.exports = {
 
     if (perQueryUnit === 'µs') perQueryStats /= 1000000;
 
-    const queryCountDetail = await I.grabTextFrom(countLocator);
+    let queryCountDetail = await I.grabTextFrom(countLocator);
```

Comment:
looks like a duplicated peace of code from line number 29

Reply:
fixed